sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0 (Unreleased)
```

Comment:
Wait to do this until the point of release

Reply:
We don't have to - when we release, we would also put the release date.